src/assets/index.tsx

```diff
@@ -1,6 +1,8 @@
 export const images = {
   dashboard: require('./images/dashboard.png'),
   dashboardInactive: require('./images/dashboardInactive.png'),
+  authenticators: require('./images/authenticatorsActive.png'),
+  authenticatorsInactive: require('./images/authenticatorsInactive.png'),
```

Comment:
Please add assets in 3 sizes

Reply:
I got only one size but Mateusz said that this icon will be probably changed in the future so I will upload 3 sizes than